be/src/exprs/json_functions.h

```diff
@@ -72,6 +72,7 @@ class JsonFunctions {
 
     /**
      * The `document` parameter must be has parsed.
+     * return Value Is Array object
```

Comment:
better to change its name to `get_json_array_from_parsed_json()` to make it more explict.

And in `src//exec/json_scanner.cpp` line 270, you can this method, but judge the return value as either array or non-array? the `else` should not happen?

Reply:
The method MUST return ArrayObject.
 `else if (!root->IsArray())` is true, then I warp a ArrayObject  in `root` outside.
`else` case  MUST be ArrayObject

Reply:
The method name changed to `get_json_objects_from_parsed_json` ??